code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -724,7 +724,7 @@
 	to_chat(user, span_notice("You offer [src] to [SM]..."))
 	being_used = TRUE
 
-	var/list/candidates = pollCandidatesForMob("Do you want to play as [SM.name]?", ROLE_SENTIENCE, null, ROLE_SENTIENCE, 50, SM, POLL_IGNORE_SENTIENCE_POTION) // see poll_ignore.dm
+	var/list/candidates = pollCandidatesForMob("Do you want to play as [SM.name]?", ROLE_SENTIENCE, null, ROLE_SENTIENCE, 50, SM, POLL_IGNORE_SENTIENCE_POTION, disallow_started_as_observer = TRUE) // see poll_ignore.dm
```

Comment:
this is the most innocent thing there is, noone even rolls for this why block it